configure-ironic.sh

```diff
@@ -45,3 +49,22 @@ EOF
 
 mkdir -p /shared/html
 mkdir -p /shared/ironic_prometheus_exporter
+
+if [ "$USE_HTTP_BASIC" = "true" ]; then
+
+crudini --merge /etc/ironic/ironic.conf <<EOF
+[DEFAULT]
+auth_strategy = http_basic
+http_basic_auth_user_file = /shared/htpasswd-ironic
+
+[json_rpc]
+auth_strategy = http_basic
+http_basic_auth_user_file = /shared/htpasswd-json-rpc
```

Comment:
lgtm but if you want you could just reuse /shared/htpasswd-ironic here.

Reply:
ack I will update